fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java

```diff
@@ -332,7 +271,13 @@ public enum ErrorCode {
      * 5500 - 5599: DDL operation failure
      */
     ERR_LOC_AWARE_UNSUPPORTED_FOR_COLOCATE_TBL(5500, new byte[] {'4', '2', '0', '0', '0'},
-            "table '%s' has location property and cannot be colocated")
+            "table '%s' has location property and cannot be colocated"),
+    ERR_BAD_DB_ERROR(5501, new byte[] {'4', '2', '0', '0', '0'}, "Unknown database '%s'"),
+    ERR_BAD_TABLE_ERROR(5502, new byte[] {'4', '2', '0', '0', '0'}, "Unknown table '%s'"),
+    ERR_NOT_OLAP_TABLE(5503, new byte[] {'4', '2', '0', '0', '0'}, "Table '%s' is not a OLAP table"),
+    ERR_MULTI_SUB_PARTITION(5504, new byte[] {'4', '2', '0', '0', '0'},
+            "Partition '%s' has sub partitions, should specify the partition id"),
```

Comment:
Sub-partition is not a user seeable concept. will it be returned to users?

Reply:
Yes, when setting the partition version of random distribution table, the partition id should be specified.